main.nf

```diff
@@ -109,6 +109,8 @@ def mergeArgs (stem, data_type, args) {
 process image_to_zarr {
     tag "${image}"
     debug verbose_log
+    label 'big_mem'
```

Comment:
I've split the label for memory and cpu to give more granular control over resource allocation.